Traceback (most recent call last):
Traceback (most recent call last):
  File "src/reward_modelling/reward_model.py", line 214, in <module>
Traceback (most recent call last):
  File "src/reward_modelling/reward_model.py", line 214, in <module>
Traceback (most recent call last):
  File "src/reward_modelling/reward_model.py", line 214, in <module>
  File "src/reward_modelling/reward_model.py", line 214, in <module>
Traceback (most recent call last):
  File "src/reward_modelling/reward_model.py", line 214, in <module>
        training_args, model_args = parser.parse_args_into_dataclasses()training_args, model_args = parser.parse_args_into_dataclasses()    

training_args, model_args = parser.parse_args_into_dataclasses()  File "/sailhome/fongsu/rm/lib/python3.8/site-packages/transformers/hf_argparser.py", line 332, in parse_args_into_dataclasses

  File "/sailhome/fongsu/rm/lib/python3.8/site-packages/transformers/hf_argparser.py", line 332, in parse_args_into_dataclasses
  File "/sailhome/fongsu/rm/lib/python3.8/site-packages/transformers/hf_argparser.py", line 332, in parse_args_into_dataclasses
    training_args, model_args = parser.parse_args_into_dataclasses()
  File "/sailhome/fongsu/rm/lib/python3.8/site-packages/transformers/hf_argparser.py", line 332, in parse_args_into_dataclasses
    training_args, model_args = parser.parse_args_into_dataclasses()
  File "/sailhome/fongsu/rm/lib/python3.8/site-packages/transformers/hf_argparser.py", line 332, in parse_args_into_dataclasses
        obj = dtype(**inputs)    obj = dtype(**inputs)
obj = dtype(**inputs)

  File "<string>", line 110, in __init__
  File "<string>", line 110, in __init__
  File "<string>", line 110, in __init__
        obj = dtype(**inputs)obj = dtype(**inputs)

  File "<string>", line 110, in __init__
  File "<string>", line 110, in __init__
  File "/sailhome/fongsu/rm/lib/python3.8/site-packages/transformers/training_args.py", line 1446, in __post_init__
  File "/sailhome/fongsu/rm/lib/python3.8/site-packages/transformers/training_args.py", line 1446, in __post_init__
  File "/sailhome/fongsu/rm/lib/python3.8/site-packages/transformers/training_args.py", line 1446, in __post_init__
  File "/sailhome/fongsu/rm/lib/python3.8/site-packages/transformers/training_args.py", line 1446, in __post_init__
  File "/sailhome/fongsu/rm/lib/python3.8/site-packages/transformers/training_args.py", line 1446, in __post_init__
        self.hf_deepspeed_config = HfTrainerDeepSpeedConfig(self.deepspeed)self.hf_deepspeed_config = HfTrainerDeepSpeedConfig(self.deepspeed)

  File "/sailhome/fongsu/rm/lib/python3.8/site-packages/transformers/deepspeed.py", line 77, in __init__
  File "/sailhome/fongsu/rm/lib/python3.8/site-packages/transformers/deepspeed.py", line 77, in __init__
    self.hf_deepspeed_config = HfTrainerDeepSpeedConfig(self.deepspeed)
  File "/sailhome/fongsu/rm/lib/python3.8/site-packages/transformers/deepspeed.py", line 77, in __init__
    self.hf_deepspeed_config = HfTrainerDeepSpeedConfig(self.deepspeed)
  File "/sailhome/fongsu/rm/lib/python3.8/site-packages/transformers/deepspeed.py", line 77, in __init__
    self.hf_deepspeed_config = HfTrainerDeepSpeedConfig(self.deepspeed)
  File "/sailhome/fongsu/rm/lib/python3.8/site-packages/transformers/deepspeed.py", line 77, in __init__
        super().__init__(config_file_or_dict)super().__init__(config_file_or_dict)
    
  File "/sailhome/fongsu/rm/lib/python3.8/site-packages/transformers/deepspeed.py", line 67, in __init__
super().__init__(config_file_or_dict)  File "/sailhome/fongsu/rm/lib/python3.8/site-packages/transformers/deepspeed.py", line 67, in __init__

  File "/sailhome/fongsu/rm/lib/python3.8/site-packages/transformers/deepspeed.py", line 67, in __init__
    super().__init__(config_file_or_dict)
  File "/sailhome/fongsu/rm/lib/python3.8/site-packages/transformers/deepspeed.py", line 67, in __init__
    super().__init__(config_file_or_dict)
  File "/sailhome/fongsu/rm/lib/python3.8/site-packages/transformers/deepspeed.py", line 67, in __init__
            super().__init__(config_file_or_dict)super().__init__(config_file_or_dict)super().__init__(config_file_or_dict)


  File "/sailhome/fongsu/rm/lib/python3.8/site-packages/accelerate/utils/deepspeed.py", line 53, in __init__
  File "/sailhome/fongsu/rm/lib/python3.8/site-packages/accelerate/utils/deepspeed.py", line 53, in __init__
  File "/sailhome/fongsu/rm/lib/python3.8/site-packages/accelerate/utils/deepspeed.py", line 53, in __init__
    super().__init__(config_file_or_dict)
  File "/sailhome/fongsu/rm/lib/python3.8/site-packages/accelerate/utils/deepspeed.py", line 53, in __init__
    super().__init__(config_file_or_dict)
  File "/sailhome/fongsu/rm/lib/python3.8/site-packages/accelerate/utils/deepspeed.py", line 53, in __init__
        config_decoded = base64.urlsafe_b64decode(config_file_or_dict).decode("utf-8")    config_decoded = base64.urlsafe_b64decode(config_file_or_dict).decode("utf-8")
config_decoded = base64.urlsafe_b64decode(config_file_or_dict).decode("utf-8")
  File "/usr/lib/python3.8/base64.py", line 133, in urlsafe_b64decode

      File "/usr/lib/python3.8/base64.py", line 133, in urlsafe_b64decode
config_decoded = base64.urlsafe_b64decode(config_file_or_dict).decode("utf-8")  File "/usr/lib/python3.8/base64.py", line 133, in urlsafe_b64decode

  File "/usr/lib/python3.8/base64.py", line 133, in urlsafe_b64decode
    config_decoded = base64.urlsafe_b64decode(config_file_or_dict).decode("utf-8")
  File "/usr/lib/python3.8/base64.py", line 133, in urlsafe_b64decode
        return b64decode(s)return b64decode(s)    
        
return b64decode(s)return b64decode(s)  File "/usr/lib/python3.8/base64.py", line 87, in b64decode
return b64decode(s)  File "/usr/lib/python3.8/base64.py", line 87, in b64decode



  File "/usr/lib/python3.8/base64.py", line 87, in b64decode
  File "/usr/lib/python3.8/base64.py", line 87, in b64decode
  File "/usr/lib/python3.8/base64.py", line 87, in b64decode
        return binascii.a2b_base64(s)return binascii.a2b_base64(s)

    return binascii.a2b_base64(s)
binasciibinascii..Error        Error: : binasciireturn binascii.a2b_base64(s)return binascii.a2b_base64(s)Invalid base64-encoded string: number of data characters (49) cannot be 1 more than a multiple of 4Invalid base64-encoded string: number of data characters (49) cannot be 1 more than a multiple of 4.



Error: binasciibinasciiInvalid base64-encoded string: number of data characters (49) cannot be 1 more than a multiple of 4
..ErrorError: : Invalid base64-encoded string: number of data characters (49) cannot be 1 more than a multiple of 4Invalid base64-encoded string: number of data characters (49) cannot be 1 more than a multiple of 4

